apis/nucleus/src/viz.js

```diff
@@ -106,27 +109,43 @@ export default function viz({ model, halo, initialError, onDestroy = async () =>
       unmountCell = noopi;
     },
     /**
-     * Converts the visualization to a different registered type
+     * Converts the visualization to a different registered type. Will update properties if permissions allow, else will patch.
+     *
+     * Not all chart types are compatible, similar structures are required.
+     *
+     * NOTE: Consider using viz.convert.toType instead for session based conversion
```

Comment:
Maybe we should wait with referencing `viz.convert` until it's part of the api?
Also would be nice to clarify why the user should consider using that instead... some specific benefit?

Reply:
yes yes, i'll remove that part